content/docs/guides/converting-gatsby-to-tina.mdx

```diff
@@ -343,22 +271,609 @@ Add the following to the top of `src/style.css`. This will resize any images in
+
+//...
+
+```
+
+```diff
++ import formatDate from "dateformat"
+//...
+      <ol style={{ listStyle: `none` }}>
+        {posts.map(post => {
+-          const title = post.frontmatter.title || post.fields.slug
++          const title = post.title || post.slug
+
+          return (
+-            <li key={post.fields.slug}>
++            <li key={post.slug}>
+              <article
+                className="post-list-item"
+                itemScope
+                itemType="http://schema.org/Article"
+              >
+                <header>
+                  <h2>
+-                    <Link to={post.fields.slug} itemProp="url">
++                    <Link to={post.slug} itemProp="url">
+                      <span itemProp="headline">{title}</span>
+                    </Link>
+                  </h2>
+-                  <small>{post.frontmatter.date}</small>
++                  <small>{post.date}</small>
+                </header>
+                <section>
+                  <p
+                    dangerouslySetInnerHTML={{
+-                      __html: post.frontmatter.description,
++                      __html: post.description,
+                    }}
+                    itemProp="description"/>
+//...
+```
+
+```diff
+- export const Head = ({ data: { mdx: post } }) => {
++ export const Head = ({ serverData }) => {
+  return (
+    <Seo
+-      title={post.title}
++      title={serverData.data.post.title}
+-      description={post.description}
++      description={serverData.data.description}
+    />
+  )
+}
+```
+
+> Note: we don't need to use the useTina hook here because the home page is static.
+
+We'll also format the date in this file. Note that we don't need to use useEffect here because the page is static.
+
+```diff
+//...
+<ol style={{ listStyle: `none` }}>
+  {posts.map(post => {
++    const formattedDate = formatDate(post.date, "mmmm mm, yyyy")
+    const title = post.title || post.slug
+
+    return (
+      <li key={post.slug}>
+        <article
+          className="post-list-item"
+          itemScope
+          itemType="http://schema.org/Article"
+        >
+          <header>
+            <h2>
+              <Link to={post.slug} itemProp="url">
+                <span itemProp="headline">{title}</span>
+              </Link>
+            </h2>
+-            <small>{post.date}</small>
++            <small>{formattedDate}</small>
+          </header>
+          <section>
+            <p
+              dangerouslySetInnerHTML={{
+                __html: post.description,
+              }}
+              itemProp="description"
+//...
+```
+
+The final thing we'll need to do to add contextual editing is to configure the routing property of our collection. This will navigate us to the correct page
+when we open the file in TinaCMS' visual editor. Because we're storing each blog post in a single folder in `content` we can just use the first
+folder in the breadcrumbs array to retrieve the correct path.
```

Comment:
```suggestion
The final step for enabling contextual editing is to configure the `routing` property of our collection. This setting will ensure that we navigate to the correct page when opening a file in TinaCMS's visual editor. Since each blog post is stored in its own folder within the `content` directory, we can use the first folder in the breadcrumbs array to determine the correct path.
```

I think this would be better if we used the `.filename` prop instead of breadcrumbs here.